vite.config.js

```
@@ -0,0 +1,64 @@
+
+// Read package.json directly to get peerDependencies
+const pkg = JSON.parse(fs.readFileSync('./package.json', 'utf8'));
+// Use require for libName since it's a CommonJS module
+const libName = require("./libName.cjs");
+
+export default defineConfig({
+  plugins: [
+    // Explicitly configure react plugin to use babel with styled-components
+    react({
+      babel: {
+        plugins: ['babel-plugin-styled-components'],
```

ESLint

[error] Replace `'babel-plugin-styled-components'],` with `"babel-plugin-styled-components"]`

(prettier/prettier)